src/test/java/Phone.java

```diff
@@ -1,23 +1,29 @@
 import java.io.PrintStream;
+import java.util.Calendar;
+import java.util.concurrent.TimeUnit;
 
 public class Phone implements Creature{
     String phoneType;
+    long battery = TimeUnit.HOURS.toMillis(1);
+    int callsCount;
+    double callReduce = (battery*2)/100; //2% from 1 hour
+
+    long currDate = System.currentTimeMillis();
+    long objectCreationDate;
 
     public Phone(String phoneType) {
         this.phoneType = phoneType;
+        this.objectCreationDate = System.currentTimeMillis();
     }
 
     public void call(String number) {
         System.out.println("Dialing - " + number);
+        callsCount++;
     }
 
-    public boolean batteryStatus(boolean res) {
-        if (res) {
-            System.out.println("Battery is full");
-        } else {
-            System.out.println("Low battery");
-        }
-        return res;
+    public long batteryPercentage() {
+        long batteryPercentage = (long) ((100*((battery - (currDate - objectCreationDate) - (callsCount * callReduce))))/battery);
```

Comment:
`currDate` has to be reloaded each time you do the calculation. Please replace it with `System.currentTimeMillis()`.